src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/ValidationError.cs

```diff
@@ -17,37 +18,20 @@ internal class ValidationError
     {
         private Type _exceptionType;
 
-        /// <summary>
-        /// Creates an instance of <see cref="ValidationError"/>
-        /// </summary>
-        /// <param name="MessageDetail"/> contains information about the exception that is used to generate the exception message.
-        /// <param name="exceptionType"/> is the type of exception that occurred.
-        /// <param name="failureType"/> is the type of validation failure that occurred.
-        /// <param name="stackFrame"/> is the stack frame where the exception occurred.
-        internal ValidationError(
-            MessageDetail MessageDetail,
-            ValidationFailureType failureType,
-            Type exceptionType,
-            StackFrame stackFrame)
-            : this(MessageDetail, failureType, exceptionType, stackFrame, null)
-        {
-            // TODO: need to include CallContext.
-        }
-
         /// <summary>
         /// Creates an instance of <see cref="ValidationError"/>
         /// </summary>
         /// <param name="messageDetail"/> contains information about the exception that is used to generate the exception message.
         /// <param name="exceptionType"/> is the type of exception that occurred.
-        /// <param name="failureType"/> is the type of validation failure that occurred.
         /// <param name="stackFrame"/> is the stack frame where the exception occurred.
+        /// <param name="failureType"/> is the type of validation failure that occurred.
         /// <param name="innerException"/> is the inner exception that occurred.
         internal ValidationError(
             MessageDetail messageDetail,
-            ValidationFailureType failureType,
             Type exceptionType,
             StackFrame stackFrame,
-            Exception innerException)
+            ValidationFailureType failureType,
```

Comment:
Be consistent with the naming.
Here we have failureType, above in LifetimeValidationError (and others) the parameter is validationFailureType.
I suggest validationFailureType.